tutorials-examples/portal-intermediate.rst

```diff
@@ -120,10 +120,10 @@ Use the settings icon (two gears at upper right) to open the plot parameters pop
     Plot parameters for the seeing versus time plot.
 
 3.4. To add a plot to visualize the astrometric scatter: use the settings icon, choose "Add New Chart" and match the parameters shown below, then click "OK".
-Note that in both the X and Y parameters, the difference between the DiaSource coordinate and the DiaObject coordinate are divided by 3600, so that the plot axes are in arcseconds: ``(ra-67.4579634)*3600`` and ``(decl+44.080243)*3600``.
+Note that in both the X and Y parameters, the difference between the DiaSource coordinate and the DiaObject coordinate are divided by 3600, so that the plot axes are in arcseconds: ``((ra-67.4579634)*cos(decl*(3.14159/180)))*3600`` and ``(decl+44.080243)*3600``.
```

Comment:
Let's use `pi()` instead of "3.14159" so that readers learn a little more about the Portal expression language.

(We'll probably provide an actual degrees-to-radians conversion function at some point, but that's likely to be a few months from now.)

Reply:
I can see that pi() has been used instead of 3.14159, so am resolving this conversation.